modules/ROOT/pages/sap/sap-s4-hana-connector.adoc

```diff
@@ -0,0 +1,466 @@
+= SAP S/4 HANA Cloud Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+
+Support Category: Premium
+
+SAP S/4 HANA Cloud Connector Version 1.0
+
+The Mule SAP S/4 HANA Cloud Connector provides full support to query, get, create, update, delete entities and execute function using the ODATA API exposed by S/4 HANA.
```

Comment:
The Mule SAP S/4HANA Cloud Connector provides full support for the query, get, create, update, delete entities, and execute functions using the ODATA API exposed by S/4HANA.